int32 ID
float64 obst_velocity
float64 obst_orientation
geometry_msgs/Point obst_pose
 